From 3b1426f7556244844a2683b9f0bc539278773742 Mon Sep 17 00:00:00 2001
From: Martin Wilck <mwilck@suse.com>
Date: Fri, 1 Mar 2019 21:08:29 +0100
Subject: [PATCH 4/6] waf: install help files into /usr/share/help

The new location is /usr/share/help/$LINGUA/$DOMAIN rather
than /usr/share/gnome/help/$DOMAIN/$LINGUA.

Signed-off-by: Martin Wilck <mwilck@suse.com>
---
 help/wscript            | 6 +++---
 wafadmin/Tools/gnome.py | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/help/wscript b/help/wscript
index f02864e..a835fb5 100644
--- a/help/wscript
+++ b/help/wscript
@@ -27,7 +27,7 @@ def init_mallard(self):
 		print "\n".join(missing_vars)
 
 	self.bld.env.update(DOC_VAR)
-	self.default_install_path='${PREFIX}/share/gnome/help/${DOC_ID}'
+	self.default_install_path='${PREFIX}/share/help'
 
 def apply_mallard(self):
 	bld = self.bld
@@ -53,11 +53,11 @@ def apply_mallard(self):
 			src = self.path.find_resource('C/%s' % page)
 			tsk.set_inputs([node,src])
 			tsk.set_outputs(out)
-			tsk.install_path = os.path.join(self.install_path, lang)
+			tsk.install_path = os.path.join(self.install_path, lang, "${DOC_ID}")
 	if bld.is_install:
 		for page in pages:
 			out = "%s/%s" % (cnode.abspath(), page)
-			instdir = os.path.join(self.install_path, "C")
+			instdir = os.path.join(self.install_path, "C", "${DOC_ID}")
 			bld.install_files(instdir, out)
 
 Task.simple_task_type('xml2po', '${XML2PO} ${XML2POFLAGS} ${SRC} > ${TGT}',
diff --git a/wafadmin/Tools/gnome.py b/wafadmin/Tools/gnome.py
index f77e9b8..ee78d19 100644
--- a/wafadmin/Tools/gnome.py
+++ b/wafadmin/Tools/gnome.py
@@ -65,7 +65,7 @@ def apply_gnome_doc(self):
 		tsk2.inputs=[node,out]
 		tsk2.run_after.append(tsk)
 		if bld.is_install:
-			path=self.install_path+'/gnome/help/%s/%s'%(self.doc_module,x)
+			path=self.install_path+'/help/%s/%s'%(x,self.doc_module)
 			bld.install_files(self.install_path+'/omf',out2,env=self.env)
 			for y in self.to_list(self.doc_figures):
 				try:
-- 
2.20.1

